assets/javascripts/discourse/routes/admin-plugins-explorer.js.es6

```diff
@@ -6,17 +6,21 @@ export default Discourse.Route.extend({
   model() {
     const p1 = this.store.findAll("query");
     const p2 = ajax("/admin/plugins/explorer/schema.json", { cache: true });
+    const p3 = ajax("/admin/plugins/explorer/groups.json"); // Should cache be true?
```

Comment:
This will not cache ... its not really a cachable route, one consideration here is preload store, may make sense to use it. or amend serializer to include groups

Reply:
Using existing group api now